_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -29,9 +29,13 @@ This rule applies to any [HTML element][] that is [visible][] and for which the
 
 For each test target, at least one of the following is true:
 
-- **not important**: the [computed][] value of its [letter-spacing][] property is not [important][]; or
-- **wide enough**: the [computed][] value of its [letter-spacing][] property is at least 0.12 times the [computed][] value of its [font-size][] property; or
-- **cascade**: the [cascaded][] value of its [letter-spacing][] property is not a value [declared][] in its `style` attribute.
+- <dfn id="24afc2:not-important">not important</dfn>: the [computed][] value of its [letter-spacing][] property is not [important][]; or
+- <dfn id="24afc2:spaced-text">spaced text</dfn>: for each text node descendant of this target, one of the following is true:
```

Comment:
> Would it be easier to change the applicability for this like the line height rule? E.g., the applicability would become
> 
> `This rule applies to any HTML element with [visible](https://act-rules.github.io/rules/78fd32#visible) text for which the style attribute [declares](https://www.w3.org/TR/css-cascade-4/#declared) the [letter-spacing](https://www.w3.org/TR/css-text-3/#propdef-letter-spacing) CSS property.`

It is the case already: [Applicability](https://github.com/act-rules/act-rules.github.io/pull/1804/files#diff-3ca6de72ad2dd5a4af825a1785d687778e21647d8311d90cf0696a3734626393R26)

> I'm not sure I completely understand what you are trying to fix here. Specifically I don't quite follow what you mean when you say `which is incorrect since both the font-size or the text spacing property can be updated before the actual text. ` in your description above.

That are the examples in issues #1687 and #1688 which this PR fixes.